src/splitInteger.test.js

```diff
@@ -8,12 +8,22 @@ describe(`Function 'splitInteger':`, () => {
   });
 
   it(`should return an array`, () => {
+    expect(splitInteger(5, 1)).toBeInstanceOf(Array);
+  });
+
+  it(`should return an array [5] if (num = 5, parts = 1)`, () => {
+    expect(splitInteger(5, 1)).toEqual([5]);
+  });
 
+  it(`should return empty array if (num = 5, parts = 0)`, () => {
+    expect(splitInteger(5, 0)).toEqual([]);
   });
```

Comment:
‘The input to your function will always be valid’, so you need to test only positive integers. This test is not needed